platform/machine/qemu/disk.go

```diff
@@ -195,3 +193,12 @@ func setupDisk(additionalOptions ...string) (*os.File, error) {
 
 	return os.OpenFile(dstFileName, os.O_RDWR, 0)
 }
+
+func mkpath(basedir string) (string, error) {
+	f, err := ioutil.TempFile(basedir, "mantle-qemu")
+	if err != nil {
+		return "", err
+	}
+	defer f.Close()
```

Comment:
This seems to have a subtle difference in functionality as this will be closed at the exit of `mkpath` rather than `setupDisk` but given the rest of the function it doesn't seem to change anything.

Reply:
`setupDisk` was closing the file immediately, not from a `defer`.